tensorbay/utility/common.py

```diff
@@ -3,14 +3,7 @@
 # Copyright 2021 Graviti. Licensed under MIT License.
 #
 
-"""Common_loads method, EqMixin class.
-
-:meth:`common_loads` is a common method for loading an object from a dict or a list of dict.
-
-:class:`EqMixin` is a mixin class to support __eq__() method,
-which compares all the instance variables.
-
-"""
+"""Related tools for common use."""
```

Comment:
what is `related tools` mean?

Reply:
It seems all three methods here are general supports for most functions throughtout the tensorbay. Neverthelss, the three of them are relatively independent and not related. So I use the vague concept `related tools` here. I'm not much sure but happy to see any other suitable word.

Reply:
All three functions/functions here are tools for common use throughout the TensorBay. Nevertheless, the three of them have weak or no relationship with each other. Hence I use a vague Concept `related tools` to represent them. At the same time, what do you think of this whole file?